apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,45 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should undo addition in my reading list', async () => {
```

Comment:
Provide proper description for all `it`.
Also move common code to `beforeEach()` to reduce file size.